.gitignore

```diff
@@ -0,0 +1,2 @@
+suomen-sanomalehtikielen-taajuussanasto-B9996.txt
```

Comment:
Not sure about copyright.
Is it OK for me to: 
- redistribute this file
- also store the explanation from wiktionary in the sqlite
- publish these here on github?